tests/collectioncardmakertest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require('../src/CollectionCardMaker.php');
+
+use PHPUnit\Framework\TestCase;
+
+class CollectionCardMakerTest extends TestCase {
+
+
+public function testcreateGameCard()
+{
+   
+    $name = 'wow';
+    $playercount = '2 - 4';
+    $difficulty = '2';
+    $description = 'blah';
+
+    $creategamecard = new CollectionCardMaker($name, $playercount, $difficulty, $description);
+    $outcome = 
```

Comment:
this variable name is a little confusing, because outcome and result mean the same thing. i would normally call them "expectedResult" and "actualResult", or expected whatever and actual whatever